Tests fail on perl > 5.26 due to . missing from @INC

Getting errors like: Can't locate t/lib/TestAppUnicode.pm in `@INC`

Full test output:
```
02:19:26 26939 ERROR [463]: + PERL_USE_UNSAFE_INC=0
02:19:26 26939 ERROR [463]: + /usr/bin/make test
02:19:26 26939 ERROR [463]: PERL_DL_NONLAZY=1 "/usr/local/cpanel/3rdparty/perl/528/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t t/00_base/*.t t/01_config/*.t t/02_request/*.t t/03_route_handler/*.t t/04_static_file/*.t t/05_views/*.t t/06_helpers/*.t t/07_apphandlers/*.t t/08_session/*.t t/09_cookies/*.t t/10_template/*.t t/11_logger/*.t t/12_response/*.t t/13_engines/*.t t/14_serializer/*.t t/15_plugins/*.t t/16_timer/*.t t/17_apps/*.t t/19_dancer/*.t t/20_deprecation/*.t t/21_dependents/*.t t/22_hooks/*.t t/23_dancer_tests/*.t t/24_deployment/*.t t/25_exceptions/*.t
02:19:26 26939 ERROR [463]: t/00-compile.t ...................................... ok
02:19:26 26939 ERROR [463]: do "t/00-report-prereqs.dd" failed, '.' is no longer in @INC; did you mean do "./t/00-report-prereqs.dd"? at t/00-report-prereqs.t line 71.
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: # Versions for all modules listed in MYMETA.json (including optional ones):
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: # === Configure Requires ===
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: #     Module              Want Have
02:19:26 26939 ERROR [463]: #     ------------------- ---- ----
02:19:26 26939 ERROR [463]: #     ExtUtils::MakeMaker  any 7.34
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: # === Configure Suggests ===
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: #     Module      Want    Have
02:19:26 26939 ERROR [463]: #     -------- ------- -------
02:19:26 26939 ERROR [463]: #     JSON::PP 2.27300 2.97001
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: # === Build Requires ===
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: #     Module              Want Have
02:19:26 26939 ERROR [463]: #     ------------------- ---- ----
02:19:26 26939 ERROR [463]: #     ExtUtils::MakeMaker  any 7.34
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: # === Test Requires ===
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: #     Module               Want     Have
02:19:26 26939 ERROR [463]: #     ------------------- ----- --------
02:19:26 26939 ERROR [463]: #     Data::Dump            any     1.23
02:19:26 26939 ERROR [463]: #     Devel::Hide           any   0.0010
02:19:26 26939 ERROR [463]: #     Digest::MD5           any     2.55
02:19:26 26939 ERROR [463]: #     ExtUtils::MakeMaker   any     7.34
02:19:26 26939 ERROR [463]: #     File::Spec            any     3.75
02:19:26 26939 ERROR [463]: #     HTTP::CookieJar     0.008    0.008
02:19:26 26939 ERROR [463]: #     HTTP::Request         any     6.18
02:19:26 26939 ERROR [463]: #     HTTP::Tiny          0.014    0.070
02:19:26 26939 ERROR [463]: #     IO::Handle            any     1.39
02:19:26 26939 ERROR [463]: #     IO::Socket::INET      any     1.39
02:19:26 26939 ERROR [463]: #     IPC::Open3            any     1.20
02:19:26 26939 ERROR [463]: #     JSON                 2.90  2.97001
02:19:26 26939 ERROR [463]: #     Plack::Builder        any    undef
02:19:26 26939 ERROR [463]: #     Test::More            any 1.302140
02:19:26 26939 ERROR [463]: #     Test::NoWarnings      any     1.04
02:19:26 26939 ERROR [463]: #     blib                 1.01     1.07
02:19:26 26939 ERROR [463]: #     utf8                  any     1.21
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: # === Test Recommends ===
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: #     Module         Want     Have
02:19:26 26939 ERROR [463]: #     ---------- -------- --------
02:19:26 26939 ERROR [463]: #     CPAN::Meta 2.120900 2.150010
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: # === Runtime Requires ===
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: #     Module                      Want     Have
02:19:26 26939 ERROR [463]: #     -------------------------- ----- --------
02:19:26 26939 ERROR [463]: #     Carp                         any     1.50
02:19:26 26939 ERROR [463]: #     Cwd                          any     3.75
02:19:26 26939 ERROR [463]: #     Data::Dumper                 any    2.170
02:19:26 26939 ERROR [463]: #     Encode                       any     2.97
02:19:26 26939 ERROR [463]: #     Exporter                     any     5.73
02:19:26 26939 ERROR [463]: #     Fcntl                        any     1.13
02:19:26 26939 ERROR [463]: #     File::Basename               any     2.85
02:19:26 26939 ERROR [463]: #     File::Copy                   any     2.33
02:19:26 26939 ERROR [463]: #     File::Path                   any     2.15
02:19:26 26939 ERROR [463]: #     File::Spec                   any     3.75
02:19:26 26939 ERROR [463]: #     File::Spec::Functions        any     3.75
02:19:26 26939 ERROR [463]: #     File::Temp                   any   0.2308
02:19:26 26939 ERROR [463]: #     File::stat                   any     1.08
02:19:26 26939 ERROR [463]: #     FindBin                      any     1.51
02:19:26 26939 ERROR [463]: #     Getopt::Long                 any     2.50
02:19:26 26939 ERROR [463]: #     HTTP::Body                   any     1.22
02:19:26 26939 ERROR [463]: #     HTTP::Date                   any     6.02
02:19:26 26939 ERROR [463]: #     HTTP::Headers                any     6.18
02:19:26 26939 ERROR [463]: #     HTTP::Server::Simple::PSGI   any     0.16
02:19:26 26939 ERROR [463]: #     HTTP::Tiny                 0.014    0.070
02:19:26 26939 ERROR [463]: #     Hash::Merge::Simple          any    0.051
02:19:26 26939 ERROR [463]: #     IO::File                     any     1.39
02:19:26 26939 ERROR [463]: #     MIME::Types                  any     2.17
02:19:26 26939 ERROR [463]: #     Module::Runtime              any    0.016
02:19:26 26939 ERROR [463]: #     POSIX                        any     1.84
02:19:26 26939 ERROR [463]: #     Pod::Usage                   any     1.69
02:19:26 26939 ERROR [463]: #     Scalar::Util                 any     1.50
02:19:26 26939 ERROR [463]: #     Test::Builder                any 1.302140
02:19:26 26939 ERROR [463]: #     Test::More                   any 1.302140
02:19:26 26939 ERROR [463]: #     Time::HiRes                  any   1.9759
02:19:26 26939 ERROR [463]: #     Try::Tiny                    any     0.30
02:19:26 26939 ERROR [463]: #     URI                          any     1.74
02:19:26 26939 ERROR [463]: #     URI::Escape                  any     3.31
02:19:26 26939 ERROR [463]: #     base                         any     2.27
02:19:26 26939 ERROR [463]: #     bytes                        any     1.06
02:19:26 26939 ERROR [463]: #     constant                     any     1.33
02:19:26 26939 ERROR [463]: #     lib                          any     0.64
02:19:26 26939 ERROR [463]: #     overload                     any     1.30
02:19:26 26939 ERROR [463]: #     parent                       any    0.236
02:19:26 26939 ERROR [463]: #     strict                       any     1.11
02:19:26 26939 ERROR [463]: #     vars                         any     1.04
02:19:26 26939 ERROR [463]: #     warnings                     any     1.42
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: # === Runtime Recommends ===
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: #     Module      Want    Have
02:19:26 26939 ERROR [463]: #     ----------- ---- -------
02:19:26 26939 ERROR [463]: #     MIME::Types 2.17    2.17
02:19:26 26939 ERROR [463]: #     YAML         any missing
02:19:26 26939 ERROR [463]: #     YAML::XS     any missing
02:19:26 26939 ERROR [463]: # 
02:19:26 26939 ERROR [463]: t/00-report-prereqs.t ............................... ok
02:19:26 26939 ERROR [463]: t/00_base/000_create_fake_env.t ..................... ok
02:19:26 26939 ERROR [463]: # Testing Dancer 1.3400, Perl 5.028000, /usr/local/cpanel/3rdparty/perl/528/bin/perl
02:19:26 26939 ERROR [463]: t/00_base/001_load.t ................................ ok
02:19:26 26939 ERROR [463]: t/00_base/002_strict_and_warnings.t ................. ok
02:19:26 26939 ERROR [463]: t/00_base/003_syntax.t .............................. ok
02:19:26 26939 ERROR [463]: t/00_base/004_args.t ................................ ok
02:19:26 26939 ERROR [463]: t/00_base/005_module_loader.t ....................... ok
02:19:26 26939 ERROR [463]: t/00_base/007_load_syntax.t ......................... ok
02:19:26 26939 ERROR [463]: t/00_base/008_export.t .............................. ok
02:19:26 26939 ERROR [463]: t/00_base/009_syntax_export.t ....................... ok
02:19:26 26939 ERROR [463]: t/00_base/010_export_script.t ....................... ok
02:19:26 26939 ERROR [463]: t/00_base/06_dancer_object.t ........................ ok
02:19:26 26939 ERROR [463]: t/00_base/08_pod_coverage_dancer.t .................. skipped: Test::Pod::Coverage 1.08 required for testing POD coverage
02:19:26 26939 ERROR [463]: t/00_base/09_load_app.t ............................. ok
02:19:26 26939 ERROR [463]: t/00_base/11_file_utils.t ........................... ok
02:19:26 26939 ERROR [463]: Can't locate t/lib/TestAppUnicode.pm in @INC (you may need to install the t::lib::TestAppUnicode module) (@INC contains: /home/rpmbuild/rpm-build/BUILD/Dancer-1.3400/t/00_base/lib /home/rpmbuild/rpm-build/BUILD/Dancer-1.3400/blib/lib /home/rpmbuild/rpm-build/BUILD/Dancer-1.3400/blib/arch /usr/local/cpanel /usr/local/cpanel/3rdparty/perl/528/lib/perl5/cpanel_lib/x86_64-linux-64int /usr/local/cpanel/3rdparty/perl/528/lib/perl5/cpanel_lib /usr/local/cpanel/3rdparty/perl/528/lib/perl5/5.28.0/x86_64-linux-64int /usr/local/cpanel/3rdparty/perl/528/lib/perl5/5.28.0 /opt/cpanel/perl5/528/site_lib/x86_64-linux-64int /opt/cpanel/perl5/528/site_lib) at t/00_base/12_utf8_charset.t line 38.
02:19:26 26939 ERROR [463]: BEGIN failed--compilation aborted at t/00_base/12_utf8_charset.t line 38.
02:19:26 26939 ERROR [463]: t/00_base/12_utf8_charset.t ......................... 
02:19:26 26939 ERROR [463]: Dubious, test returned 2 (wstat 512, 0x200)
02:19:26 26939 ERROR [463]: No subtests run 
02:19:26 26939 ERROR [463]: t/00_base/13_dancer_singleton.t ..................... ok
02:19:26 26939 ERROR [463]: t/00_base/15_dependent_modules.t .................... skipped: Author tests not required for installation
02:19:26 26939 ERROR [463]: t/00_base/17_globalwarnings_config_on.t ............. ok
02:19:26 26939 ERROR [463]: t/00_base/config.t .................................. ok
02:19:26 26939 ERROR [463]: t/00_base/normalize_path.t .......................... ok
02:19:26 26939 ERROR [463]: # Test::TCP 2.19 loaded successfully
02:19:26 26939 ERROR [463]: # Test::More 1.302140 loaded successfully
02:19:26 26939 ERROR [463]: # JSON 2.97001 loaded successfully
02:19:26 26939 ERROR [463]: # YAML is not available
02:19:26 26939 ERROR [463]: # Clone 0.39 loaded successfully
02:19:26 26939 ERROR [463]: # Plack 1.0047 loaded successfully
02:19:26 26939 ERROR [463]: # XML::Simple is not available
02:19:26 26939 ERROR [463]: # HTTP::Parser::XS is not available
02:19:26 26939 ERROR [463]: t/00_base/optional-module-versions.t ................ ok
02:19:26 26939 ERROR [463]: t/00_base/uri_for.t ................................. ok
02:19:26 26939 ERROR [463]: t/01_config/01_settings.t ........................... ok
02:19:26 26939 ERROR [463]: t/01_config/02_mime_type.t .......................... ok
02:19:26 26939 ERROR [463]: t/01_config/03_logger.t ............................. ok
02:19:26 26939 ERROR [463]: t/01_config/04_config_file.t ........................ skipped: YAML or YAML::XS needed to run these tests
02:19:26 26939 ERROR [463]: t/01_config/05_serializers.t ........................ ok
02:19:26 26939 ERROR [463]: t/01_config/06_config_api.t ......................... skipped: YAML or YAML::XS needed to run these tests
02:19:26 26939 ERROR [463]: t/01_config/06_stack_trace.t ........................ skipped: tests require module 'Template' to be installed
02:19:26 26939 ERROR [463]: t/01_config/07_strict_config.t ...................... skipped: YAML or YAML::XS needed to run these tests
02:19:26 26939 ERROR [463]: t/01_config/08_environments.t ....................... skipped: YAML or YAML::XS needed to run these tests
02:19:26 26939 ERROR [463]: Use of uninitialized value $module in concatenation (.) or string at /home/rpmbuild/rpm-build/BUILD/Dancer-1.3400/blib/lib/Dancer/Config.pm line 234.
02:19:26 26939 ERROR [463]: t/01_config/yaml_dependency.t ....................... ok
02:19:26 26939 ERROR [463]: t/02_request/000_create_fake_env.t .................. ok
02:19:26 26939 ERROR [463]: t/02_request/01_load.t .............................. ok
02:19:26 26939 ERROR [463]: t/02_request/04_custom.t ............................ ok
02:19:26 26939 ERROR [463]: t/02_request/04_forward.t ........................... ok
02:19:26 26939 ERROR [463]: t/02_request/05_cgi_pm_compat.t ..................... ok
02:19:26 26939 ERROR [463]: t/02_request/06_init_env.t .......................... ok
02:19:26 26939 ERROR [463]: # OK
02:19:26 26939 ERROR [463]: t/02_request/07_raw_data.t .......................... ok
02:19:26 26939 ERROR [463]: t/02_request/08_param_array.t ....................... ok
02:19:26 26939 ERROR [463]: t/02_request/08_params.t ............................ ok
02:19:26 26939 ERROR [463]: t/02_request/10_mixed_params.t ...................... ok
02:19:26 26939 ERROR [463]: t/02_request/11_accessors.t ......................... ok
02:19:26 26939 ERROR [463]: t/02_request/12_base.t .............................. ok
02:19:26 26939 ERROR [463]: t/02_request/13_ajax.t .............................. ok
02:19:26 26939 ERROR [463]: t/02_request/14_uploads.t ........................... ok
02:19:26 26939 ERROR [463]: t/02_request/15_headers.t ........................... ok
02:19:26 26939 ERROR [463]: t/02_request/16_delete.t ............................ ok
02:19:26 26939 ERROR [463]: t/02_request/17_uri_base.t .......................... ok
02:19:26 26939 ERROR [463]: t/02_request/18_param_accessor.t .................... ok
02:19:26 26939 ERROR [463]: t/02_request/19_json_body.t ......................... ok
02:19:26 26939 ERROR [463]: # Selected 127.0.0.11:49850
02:19:26 26939 ERROR [463]: t/02_request/20_body.t .............................. ok
02:19:26 26939 ERROR [463]: t/02_request/21_dancer_response_multiple_params.t ... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/01_http_methods.t ................ ok
02:19:26 26939 ERROR [463]: t/03_route_handler/02_params.t ...................... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/03_routes_api.t .................. ok
02:19:26 26939 ERROR [463]: t/03_route_handler/04_routes_matching.t ............. ok
02:19:26 26939 ERROR [463]: t/03_route_handler/04_wildcards_megasplat.t ......... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/05_filter.t ...................... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/05_unicode.t ..................... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/06_redirect.t .................... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/07_compilation_warning.t ......... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/08_errors.t ...................... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/12_response.t .................... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/12_response_halt.t ............... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/14_options.t ..................... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/15_prefix.t ...................... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/16_caching.t ..................... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/16_embedded_prefixes.t ........... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/18_auto_page.t ................... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/21_ajax.t ........................ ok
02:19:26 26939 ERROR [463]: t/03_route_handler/23_filter_error_catching.t ....... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/24_multiple_params.t ............. ok
02:19:26 26939 ERROR [463]: t/03_route_handler/24_named_captures.t .............. ok
02:19:26 26939 ERROR [463]: t/03_route_handler/28_plack_mount.t ................. ok
02:19:26 26939 ERROR [463]: t/03_route_handler/29_forward.t ..................... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/29_redirect_immediately.t ........ ok
02:19:26 26939 ERROR [463]: t/03_route_handler/30_forward_session.t ............. ok
02:19:26 26939 ERROR [463]: t/03_route_handler/31_infinite_loop.t ............... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/33_vars.t ........................ ok
02:19:26 26939 ERROR [463]: t/03_route_handler/34_forward_body_post.t ........... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/35_no_further_routes.t ........... ok
02:19:26 26939 ERROR [463]: t/03_route_handler/36_false_routes.t ................ ok
02:19:26 26939 ERROR [463]: t/03_route_handler/99_bugs.t ........................ ok
02:19:26 26939 ERROR [463]: t/04_static_file/001_base.t ......................... ok
02:19:26 26939 ERROR [463]: t/04_static_file/003_mime_types_reinit.t ............ ok
02:19:26 26939 ERROR [463]: t/04_static_file/01_mime_types.t .................... ok
02:19:26 26939 ERROR [463]: t/04_static_file/02_dir_traversal.t ................. ok
02:19:26 26939 ERROR [463]: t/05_views/002_view_rendering.t ..................... ok
02:19:26 26939 ERROR [463]: t/05_views/03_layout.t .............................. skipped: Template is needed to run this tests
02:19:26 26939 ERROR [463]: t/06_helpers/000_create_fake_env.t .................. ok
02:19:26 26939 ERROR [463]: t/06_helpers/01_send_file.t ......................... ok
02:19:26 26939 ERROR [463]: t/06_helpers/02_http_status.t ....................... ok
02:19:26 26939 ERROR [463]: t/06_helpers/03_content_type.t ...................... ok
02:19:26 26939 ERROR [463]: t/06_helpers/04_status.t ............................ ok
02:19:26 26939 ERROR [463]: t/06_helpers/05_send_error.t ........................ ok
02:19:26 26939 ERROR [463]: Can't locate t/06_helpers/routes.pl in @INC (@INC contains: /home/rpmbuild/rpm-build/BUILD/Dancer-1.3400/lib /home/rpmbuild/rpm-build/BUILD/Dancer-1.3400/blib/lib /home/rpmbuild/rpm-build/BUILD/Dancer-1.3400/blib/arch /usr/local/cpanel /usr/local/cpanel/3rdparty/perl/528/lib/perl5/cpanel_lib/x86_64-linux-64int /usr/local/cpanel/3rdparty/perl/528/lib/perl5/cpanel_lib /usr/local/cpanel/3rdparty/perl/528/lib/perl5/5.28.0/x86_64-linux-64int /usr/local/cpanel/3rdparty/perl/528/lib/perl5/5.28.0 /opt/cpanel/perl5/528/site_lib/x86_64-linux-64int /opt/cpanel/perl5/528/site_lib) at /home/rpmbuild/rpm-build/BUILD/Dancer-1.3400/blib/lib/Dancer.pm line 170.
02:19:26 26939 ERROR [463]: # Looks like your test exited with 2 just after 1.
02:19:26 26939 ERROR [463]: t/06_helpers/06_load.t .............................. 
02:19:26 26939 ERROR [463]: Dubious, test returned 2 (wstat 512, 0x200)
02:19:26 26939 ERROR [463]: All 1 subtests passed 
02:19:26 26939 ERROR [463]: t/07_apphandlers/000_create_fake_env.t .............. ok
02:19:26 26939 ERROR [463]: t/07_apphandlers/01_base.t .......................... ok
02:19:26 26939 ERROR [463]: t/07_apphandlers/02_apache2_plack.t ................. ok
02:19:26 26939 ERROR [463]: t/07_apphandlers/03_psgi_app.t ...................... ok
02:19:26 26939 ERROR [463]: t/07_apphandlers/04_standalone_app.t ................ ok
02:19:26 26939 ERROR [463]: t/07_apphandlers/05_middlewares.t ................... ok
02:19:26 26939 ERROR [463]: t/07_apphandlers/05_psgi_api.t ...................... ok
02:19:26 26939 ERROR [463]: t/07_apphandlers/06_debug.t ......................... skipped: Test::Output is needed for this test
02:19:26 26939 ERROR [463]: t/07_apphandlers/07_middleware_map.t ................ ok
02:19:26 26939 ERROR [463]: t/07_apphandlers/08_is_text.t ....................... ok
02:19:26 26939 ERROR [463]: t/08_session/000_create_fake_env.t .................. ok
02:19:26 26939 ERROR [463]: t/08_session/01_load.t .............................. ok
02:19:26 26939 ERROR [463]: t/08_session/02_dependency_check.t .................. ok
02:19:26 26939 ERROR [463]: t/08_session/03_http_requests.t ..................... skipped: YAML is needed to run this test
02:19:26 26939 ERROR [463]: t/08_session/04_api.t ............................... ok
02:19:26 26939 ERROR [463]: t/08_session/05_yaml.t .............................. skipped: need YAML
02:19:26 26939 ERROR [463]: t/08_session/06_abstract.t .......................... ok
02:19:26 26939 ERROR [463]: t/08_session/07_session_expires.t ................... skipped: YAML is needed for this test
02:19:26 26939 ERROR [463]: t/08_session/08_simple.t ............................ ok
02:19:26 26939 ERROR [463]: t/08_session/09_session.t ........................... ok
02:19:26 26939 ERROR [463]: t/08_session/10_filter.t ............................ ok
02:19:26 26939 ERROR [463]: t/08_session/11_session_secure.t .................... ok
02:19:26 26939 ERROR [463]: t/08_session/12_session_name.t ...................... ok
02:19:26 26939 ERROR [463]: t/08_session/13_session_httponly.t .................. skipped: YAML is needed for this test
02:19:26 26939 ERROR [463]: t/08_session/14_session_domain.t .................... ok
02:19:26 26939 ERROR [463]: t/09_cookies/000_create_fake_env.t .................. ok
02:19:26 26939 ERROR [463]: t/09_cookies/01_use.t ............................... ok
02:19:26 26939 ERROR [463]: t/09_cookies/02_cookie_object.t ..................... ok
02:19:26 26939 ERROR [463]: t/09_cookies/03_persistence.t ....................... ok
02:19:26 26939 ERROR [463]: t/09_cookies/04_secure.t ............................ ok
02:19:26 26939 ERROR [463]: t/09_cookies/05_api.t ............................... ok
02:19:26 26939 ERROR [463]: t/09_cookies/06_expires.t ........................... ok
02:19:26 26939 ERROR [463]: t/10_template/000_create_fake_env.t ................. ok
02:19:26 26939 ERROR [463]: t/10_template/01_factory.t .......................... skipped: Template needed
02:19:26 26939 ERROR [463]: t/10_template/02_abstract_class.t ................... ok
02:19:26 26939 ERROR [463]: t/10_template/03_simple.t ........................... ok
02:19:26 26939 ERROR [463]: t/10_template/05_template_toolkit.t ................. skipped: need Template to run this test
02:19:26 26939 ERROR [463]: t/10_template/05_template_toolkit_fromdata.t ........ skipped: module Template::Provider::FromDATA required
02:19:26 26939 ERROR [463]: t/10_template/extension.t ........................... ok
02:19:26 26939 ERROR [463]: t/10_template/template.t ............................ ok
02:19:26 26939 ERROR [463]: t/11_logger/000_create_fake_env.t ................... ok
02:19:26 26939 ERROR [463]: t/11_logger/01_abstract.t ........................... ok
02:19:26 26939 ERROR [463]: Can't locate t/lib/TestUtils.pm in @INC (you may need to install the t::lib::TestUtils module) (@INC contains: /home/rpmbuild/rpm-build/BUILD/Dancer-1.3400/blib/lib /home/rpmbuild/rpm-build/BUILD/Dancer-1.3400/blib/arch /usr/local/cpanel /usr/local/cpanel/3rdparty/perl/528/lib/perl5/cpanel_lib/x86_64-linux-64int /usr/local/cpanel/3rdparty/perl/528/lib/perl5/cpanel_lib /usr/local/cpanel/3rdparty/perl/528/lib/perl5/5.28.0/x86_64-linux-64int /usr/local/cpanel/3rdparty/perl/528/lib/perl5/5.28.0 /opt/cpanel/perl5/528/site_lib/x86_64-linux-64int /opt/cpanel/perl5/528/site_lib) at t/11_logger/02_factory.t line 5.
02:19:26 26939 ERROR [463]: BEGIN failed--compilation aborted at t/11_logger/02_factory.t line 5.
02:19:26 26939 ERROR [463]: t/11_logger/02_factory.t ............................ 
02:19:26 26939 ERROR [463]: Dubious, test returned 2 (wstat 512, 0x200)
02:19:26 26939 ERROR [463]: No subtests run 
02:19:26 26939 ERROR [463]: Can't locate t/lib/TestUtils.pm in @INC (you may need to install the t::lib::TestUtils module) (@INC contains: /home/rpmbuild/rpm-build/BUILD/Dancer-1.3400/blib/lib /home/rpmbuild/rpm-build/BUILD/Dancer-1.3400/blib/arch /usr/local/cpanel /usr/local/cpanel/3rdparty/perl/528/lib/perl5/cpanel_lib/x86_64-linux-64int /usr/local/cpanel/3rdparty/perl/528/lib/perl5/cpanel_lib /usr/local/cpanel/3rdparty/perl/528/lib/perl5/5.28.0/x86_64-linux-64int /usr/local/cpanel/3rdparty/perl/528/lib/perl5/5.28.0 /opt/cpanel/perl5/528/site_lib/x86_64-linux-64int /opt/cpanel/perl5/528/site_lib) at t/11_logger/03_file.t line 6.
02:19:26 26939 ERROR [463]: BEGIN failed--compilation aborted at t/11_logger/03_file.t line 6.
02:19:26 26939 ERROR [463]: t/11_logger/03_file.t ............................... 
02:19:26 26939 ERROR [463]: Dubious, test returned 2 (wstat 512, 0x200)
02:19:26 26939 ERROR [463]: No subtests run 
02:19:26 26939 ERROR [463]: t/11_logger/04_console.t ............................ skipped: Test::Output is needed for this test
02:19:26 26939 ERROR [463]: t/11_logger/05_format.t ............................. ok
02:19:26 26939 ERROR [463]: t/11_logger/06_null.t ............................... ok
02:19:26 26939 ERROR [463]: t/11_logger/07_diag.t ............................... ok
02:19:26 26939 ERROR [463]: t/11_logger/08_serialize.t .......................... skipped: Test::Output is needed for this test
02:19:26 26939 ERROR [463]: t/11_logger/09_capture.t ............................ ok
02:19:26 26939 ERROR [463]: t/11_logger/10_note.t ............................... ok
02:19:26 26939 ERROR [463]: t/11_logger/11_runtime_file.t ....................... ok
02:19:26 26939 ERROR [463]: t/11_logger/unicode.t ............................... ok
02:19:26 26939 ERROR [463]: t/12_response/000_create_fake_env.t ................. ok
02:19:26 26939 ERROR [463]: t/12_response/01_CRLF_injection.t ................... ok
02:19:26 26939 ERROR [463]: t/12_response/02_headers.t .......................... ok
02:19:26 26939 ERROR [463]: t/12_response/03_charset.t .......................... ok
02:19:26 26939 ERROR [463]: Wide character in print at /usr/local/cpanel/3rdparty/perl/528/lib/perl5/cpanel_lib/HTTP/Server/Simple/PSGI.pm line 132, <DATA> line 16.
02:19:26 26939 ERROR [463]: Wide character in print at /usr/local/cpanel/3rdparty/perl/528/lib/perl5/cpanel_lib/HTTP/Server/Simple/PSGI.pm line 132, <DATA> line 16.
02:19:26 26939 ERROR [463]: t/12_response/04_charset_server.t ................... ok
02:19:26 26939 ERROR [463]: t/12_response/05_api.t .............................. ok
02:19:26 26939 ERROR [463]: t/12_response/06_filter_halt_status.t ............... ok
02:19:26 26939 ERROR [463]: t/12_response/07_cookies.t .......................... ok
02:19:26 26939 ERROR [463]: t/12_response/08_drop_content.t ..................... ok
02:19:26 26939 ERROR [463]: t/12_response/09_headers_to_array.t ................. ok
02:19:26 26939 ERROR [463]: Data exceeding 100 levels, truncating
02:19:26 26939 ERROR [463]: t/12_response/10_error_dumper.t ..................... ok
02:19:26 26939 ERROR [463]: Devel::Hide hides Clone.pm
02:19:26 26939 ERROR [463]: t/12_response/10_error_dumper_without_clone.t ....... ok
02:19:26 26939 ERROR [463]: t/12_response/11_CVE-2012-5572.t .................... ok
02:19:26 26939 ERROR [463]: t/13_engines/00_load.t .............................. ok
02:19:26 26939 ERROR [463]: t/13_engines/02_template_init.t ..................... ok
02:19:26 26939 ERROR [463]: t/14_serializer/01_helpers.t ........................ ok
02:19:26 26939 ERROR [463]: t/14_serializer/02_request_json.t ................... ok
02:19:26 26939 ERROR [463]: t/14_serializer/03_request_yaml.t ................... ok
02:19:26 26939 ERROR [463]: t/14_serializer/04_request_xml.t .................... ok
02:19:26 26939 ERROR [463]: t/14_serializer/05_request_mutable.t ................ skipped: YAML or YAML::XS is needed to run this test
02:19:26 26939 ERROR [463]: t/14_serializer/06_api.t ............................ ok
02:19:26 26939 ERROR [463]: t/14_serializer/07_request_jsonp.t .................. ok
02:19:26 26939 ERROR [463]: t/14_serializer/17_clear_serializer.t ............... ok
02:19:26 26939 ERROR [463]: t/14_serializer/18_mutable_template_or_serialize.t .. ok
02:19:26 26939 ERROR [463]: t/14_serializer/99_bugs.t ........................... ok
02:19:26 26939 ERROR [463]: t/15_plugins/000_create_fake_env.t .................. ok
02:19:26 26939 ERROR [463]: t/15_plugins/01_register.t .......................... ok
02:19:26 26939 ERROR [463]: t/15_plugins/02_config.t ............................ skipped: YAML or YAML::XS is needed for this test
02:19:26 26939 ERROR [463]: t/15_plugins/03_namespace.t ......................... ok
02:19:26 26939 ERROR [463]: t/15_plugins/04_apps_and_plugins.t .................. ok
02:19:26 26939 ERROR [463]: t/15_plugins/05_keywords.t .......................... ok
02:19:26 26939 ERROR [463]: t/15_plugins/05_plugins_and_OO.t .................... ok
02:19:26 26939 ERROR [463]: t/15_plugins/05b_plugins_and_c3.t ................... ok
02:19:26 26939 ERROR [463]: Can't locate t/lib/Hookee.pm in @INC (you may need to install the t::lib::Hookee module) (@INC contains: /home/rpmbuild/rpm-build/BUILD/Dancer-1.3400/lib t/lib /home/rpmbuild/rpm-build/BUILD/Dancer-1.3400/blib/lib /home/rpmbuild/rpm-build/BUILD/Dancer-1.3400/blib/arch /usr/local/cpanel /usr/local/cpanel/3rdparty/perl/528/lib/perl5/cpanel_lib/x86_64-linux-64int /usr/local/cpanel/3rdparty/perl/528/lib/perl5/cpanel_lib /usr/local/cpanel/3rdparty/perl/528/lib/perl5/5.28.0/x86_64-linux-64int /usr/local/cpanel/3rdparty/perl/528/lib/perl5/5.28.0 /opt/cpanel/perl5/528/site_lib/x86_64-linux-64int /opt/cpanel/perl5/528/site_lib) at t/15_plugins/06_hook.t line 32.
02:19:26 26939 ERROR [463]: BEGIN failed--compilation aborted at t/15_plugins/06_hook.t line 32.
02:19:26 26939 ERROR [463]: t/15_plugins/06_hook.t .............................. 
02:19:26 26939 ERROR [463]: Dubious, test returned 2 (wstat 512, 0x200)
02:19:26 26939 ERROR [463]: No subtests run 
02:19:26 26939 ERROR [463]: t/15_plugins/07_ajax_plack_builder.t ................ ok
02:19:26 26939 ERROR [463]: t/16_timer/00_base.t ................................ ok
02:19:26 26939 ERROR [463]: t/17_apps/000_create_fake_env.t ..................... ok
02:19:26 26939 ERROR [463]: t/17_apps/00_base.t ................................. ok
02:19:26 26939 ERROR [463]: t/17_apps/01_settings.t ............................. ok
02:19:26 26939 ERROR [463]: t/17_apps/02_load_app.t ............................. ok
02:19:26 26939 ERROR [463]: t/17_apps/03_prefix.t ............................... ok
02:19:26 26939 ERROR [463]: t/17_apps/05_api.t .................................. ok
02:19:26 26939 ERROR [463]: t/19_dancer/01_script.t ............................. ok
02:19:26 26939 ERROR [463]: t/19_dancer/02_script_version_from.t ................ ok
02:19:26 26939 ERROR [463]: t/20_deprecation/01_api.t ........................... ok
02:19:26 26939 ERROR [463]: t/21_dependents/Dancer-Session-Cookie.t ............. skipped: Dancer::Session::Cookie 0.14 required
02:19:26 26939 ERROR [463]: t/22_hooks/00_syntax.t .............................. ok
02:19:26 26939 ERROR [463]: t/22_hooks/01_api.t ................................. ok
02:19:26 26939 ERROR [463]: t/22_hooks/02_before.t .............................. ok
02:19:26 26939 ERROR [463]: t/22_hooks/03_after.t ............................... ok
02:19:26 26939 ERROR [463]: t/22_hooks/04_template.t ............................ ok
02:19:26 26939 ERROR [463]: t/22_hooks/05_layout.t .............................. ok
02:19:26 26939 ERROR [463]: t/22_hooks/06_serializer.t .......................... ok
02:19:26 26939 ERROR [463]: t/22_hooks/07_file.t ................................ ok
02:19:26 26939 ERROR [463]: t/22_hooks/08_error.t ............................... ok
02:19:26 26939 ERROR [463]: t/22_hooks/09_before_error_init.t ................... ok
02:19:26 26939 ERROR [463]: t/22_hooks/10_error_in_hook.t ....................... ok
02:19:26 26939 ERROR [463]: t/22_hooks/11_error_in_hook.t ....................... ok
02:19:26 26939 ERROR [463]: t/23_dancer_tests/01_basic.t ........................ ok
02:19:26 26939 ERROR [463]: t/23_dancer_tests/02_tests_functions.t .............. ok
02:19:26 26939 ERROR [463]: t/23_dancer_tests/03_uris.t ......................... ok
02:19:26 26939 ERROR [463]: t/24_deployment/01_multi_webapp.t ................... ok
02:19:26 26939 ERROR [463]: t/25_exceptions/01_exceptions.t ..................... ok
02:19:26 26939 ERROR [463]: t/25_exceptions/02_exceptions.t ..................... ok
02:19:26 26939 ERROR [463]: t/25_exceptions/03_exceptions.t ..................... ok
02:19:26 26939 ERROR [463]: t/25_exceptions/04_exceptions_warn.t ................ ok
02:19:26 26939 ERROR [463]: t/author-pod-syntax.t ............................... skipped: these tests are for testing by the author
02:19:26 26939 ERROR [463]: t/pod.t ............................................. skipped: Author tests not required for installation
02:19:26 26939 ERROR [463]: t/release-distmeta.t ................................ skipped: these tests are for release candidate testing
02:19:26 26939 ERROR [463]: 
02:19:26 26939 ERROR [463]: Test Summary Report
02:19:26 26939 ERROR [463]: -------------------
02:19:26 26939 ERROR [463]: t/00_base/12_utf8_charset.t                       (Wstat: 512 Tests: 0 Failed: 0)
02:19:26 26939 ERROR [463]:   Non-zero exit status: 2
02:19:26 26939 ERROR [463]:   Parse errors: No plan found in TAP output
02:19:26 26939 ERROR [463]: t/06_helpers/06_load.t                            (Wstat: 512 Tests: 1 Failed: 0)
02:19:26 26939 ERROR [463]:   Non-zero exit status: 2
02:19:26 26939 ERROR [463]: t/11_logger/02_factory.t                          (Wstat: 512 Tests: 0 Failed: 0)
02:19:26 26939 ERROR [463]:   Non-zero exit status: 2
02:19:26 26939 ERROR [463]:   Parse errors: No plan found in TAP output
02:19:26 26939 ERROR [463]: t/11_logger/03_file.t                             (Wstat: 512 Tests: 0 Failed: 0)
02:19:26 26939 ERROR [463]:   Non-zero exit status: 2
02:19:26 26939 ERROR [463]:   Parse errors: No plan found in TAP output
02:19:26 26939 ERROR [463]: t/15_plugins/06_hook.t                            (Wstat: 512 Tests: 0 Failed: 0)
02:19:26 26939 ERROR [463]:   Non-zero exit status: 2
02:19:26 26939 ERROR [463]:   Parse errors: No plan found in TAP output
02:19:26 26939 ERROR [463]: Files=221, Tests=1886, 48 wallclock secs ( 0.94 usr  0.32 sys + 41.21 cusr  5.16 csys = 47.63 CPU)
02:19:26 26939 ERROR [463]: Result: FAIL
02:19:26 26939 ERROR [463]: Failed 5/221 test programs. 0/1886 subtests failed.
02:19:26 26939 ERROR [463]: make: *** [test_dynamic] Error 255
```

--- a/t/00_base/12_utf8_charset.t
+++ b/t/00_base/12_utf8_charset.t
@@ -7,6 +7,9 @@ use Test::More import => ['!pass'];
 use Dancer::ModuleLoader;
 use HTTP::Tiny;
 
+use FindBin qw($Bin);
+use lib "$Bin/../../";
+
 plan skip_all => "skip test with Test::TCP in win32/cygwin" if ($^O eq 'MSWin32' or $^O eq 'cygwin');
 plan skip_all => "Test::TCP is needed for this test"
     unless Dancer::ModuleLoader->load("Test::TCP" => '1.30');

--- a/t/06_helpers/06_load.t
+++ b/t/06_helpers/06_load.t
@@ -3,6 +3,9 @@ use Test::More 'no_plan', import => ['!pass'];
 use Dancer ':syntax';
 use Dancer::Test;
 
+use FindBin qw($Bin);
+use lib "$Bin/../../";
+
 my $routes = path('t', '06_helpers', 'routes.pl');
 
 ok( -f $routes, "file routes is found");

--- a/t/11_logger/02_factory.t
+++ b/t/11_logger/02_factory.t
@@ -2,6 +2,10 @@ use Test::More import => ['!pass'];
 
 use strict;
 use warnings;
+
+use FindBin qw($Bin);
+use lib "$Bin/../../";
+
 use t::lib::TestUtils;
 
 use Dancer ':syntax';

--- a/t/11_logger/03_file.t
+++ b/t/11_logger/03_file.t
@@ -3,6 +3,9 @@ use Test::More import => ['!pass'];
 use strict;
 use warnings;
 
+use FindBin qw($Bin);
+use lib "$Bin/../../";
+
 use t::lib::TestUtils;
 use Dancer;
 

--- a/t/15_plugins/06_hook.t
+++ b/t/15_plugins/06_hook.t
@@ -5,6 +5,8 @@ use Dancer ':syntax';
 use Dancer::Test;
 
 use lib 't/lib';
+use FindBin qw($Bin);
+use lib "$Bin/../../";
 
 plan tests => 6;
 

